core/runtime/jsrealm.rs

```diff
@@ -225,32 +203,25 @@ impl JsRealm {
     scope: &mut v8::HandleScope,
   ) -> Rc<ContextState> {
     let context = scope.get_current_context();
-    // SAFETY: slot is valid and set during realm creation
-    unsafe {
-      let rc = context
-        .get_aligned_pointer_from_embedder_data(CONTEXT_STATE_SLOT_INDEX);
-      let rc = &*(rc as *const Rc<ContextState>);
```

Comment:
It seemed to inconsistently not using a box here?

Reply:
Yeah, that looks like it

Reply:
That is because its a pointer to the Rc we are referencing.

`Box<Rc<ContextState>>::into_raw == *const Rc<ContextState>`